dnSpy/Tabs/TabItemImpl.cs

```diff
@@ -116,7 +116,16 @@ sealed class TheHeader : ViewModelBase {
 		protected override void OnMouseDown(MouseButtonEventArgs e) {
 			base.OnMouseDown(e);
 			if (!e.Handled) {
-				tabGroup.SetFocus(TabContent);
+				var tabItem = e.Source as TabItemImpl;
+				if (tabItem != null && e.ChangedButton == MouseButton.Middle)
+				{
+				    this.Owner.Close(tabItem);
+				}
+				else
+				{
+				    tabGroup.SetFocus(TabContent);
+				}
+
```

Comment:
Braces aren't used if there's only one statement, and if used, the opening brace is on the same line as the `if`.